pkg/auth/login.go

```diff
@@ -0,0 +1,102 @@
+package auth
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	buildahcli "github.com/containers/buildah/pkg/cli"
```

Comment:
Is there a way we can do without this.  We don't want to suck buildah into this, if we are going to move it to common.